docs/conf/portland/2021/news/events-and-activities.rst

```diff
@@ -0,0 +1,65 @@
+:template: {{year}}/generic.html
+
+.. post:: Apr 12, 2021
+   :tags: {{year}}, portland-{{year}}, tickets
+
+Get Ready for the Conference!
+=============================
+
+Hey everyone!
+
+We're just a couple of weeks away from the start of the conference, and we're getting excited to see you soon at our online venue for 3 days of content, collaboration, and conversation. We're happy to share with you some useful resources to help you prepare for the conference.
+
+Ticket sales closing on April 22nd 2021
+---------------------------------------
+
+To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **April 22nd, 2021**.
+If you haven't purchased your ticket yet we invite you to `get your tickets soon <https://www.writethedocs.org/conf/portland/2020/tickets/>`_.
+
+For those of you who already purchased your tickets, you will get an email through Ti.to with the Hopin registration link and instructions on April 19th, 2021. 
```

Comment:
We should mention that we’ll send another email and the date of that email too. 

Like “we will send an email with the login instructions on April 19 and again to all registered users on April 22”